.github/workflows/e2e-test-daily.yml

```diff
@@ -116,20 +118,17 @@ jobs:
         with:
           kubeconfig: ${{ steps.e2e_test.outputs.kubeconfig }}
 
-      - name: Notify teams channel
-        if: failure() && github.ref == 'refs/heads/main'
+      - name: Notify about failure
+        if: failure() # && github.ref == 'refs/heads/main'
         continue-on-error: true
-        shell: bash
-        working-directory: .github/actions/e2e_test
-        run: |
-          sudo apt-get install gettext-base -y
-          export TEAMS_JOB_NAME=${{ matrix.provider }}
-          export TEAMS_RUN_ID=${{ github.run_id }}
-          envsubst < teams-payload.json > to-be-send.json
-          curl                                          \
-            -H "Content-Type: application/json"         \
-            -d @to-be-send.json                         \
-            "${{ secrets.MS_TEAMS_WEBHOOK_URI }}"
+        uses: ./.github/actions/notify_failure
+        with:
+          projectWriteToken: ${{ secrets.PROJECT_WRITE_TOKEN }}
+          teamsWebhookUri: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
+          refStream: ${{ matrix.refStream }}
+          test: "sonobuoy full"
```

Comment:
Should we add `"sonobuoy full"` as another dimension in the matrix just to pass the same variable to the actual test action as we do for the notification?